code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -519,3 +519,44 @@
 	visor_flags_inv = HIDEFACE
 	adjusted_flags = null
 	actions_types = list(/datum/action/item_action/adjust)
+
+
+//Phoenix Fluff Here
+/obj/item/clothing/mask/mummy/bibledog
+	name = "sterilised facewrap"
+	desc = "Sterilised bandages fashioned into a facemask"
+	icon_state = "mummy_maskbc"
+	item_state = "mummy_maskbc"
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	flags_inv = HIDEHAIR|HIDEFACIALHAIR|HIDEEARS|HIDESNOUT	
+	visor_flags_inv = HIDEFACE
+	visor_flags_cover = MASKCOVERSMOUTH
+	gas_transfer_coefficient = 0.9
+	permeability_coefficient = 0.01
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 25, "rad" = 0, "fire" = 30, "acid" = 10, "wound" = 30)//Literally wearing bandages for mask
+	actions_types = list(/datum/action/item_action/adjust)
+
+/obj/item/clothing/mask/mummy/bibledog/attack_self(mob/user)
+	adjustmask(user)
+
+/obj/item/clothing/mask/mummy/bibledog/Initialize()
+	. = ..()
+	AddComponent(/datum/component/anti_magic, TRUE, TRUE, TRUE, ITEM_SLOT_MASK, INFINITY, FALSE)
+
+/obj/item/clothing/mask/gelatoscreen
+	name = "\improper Gelato Face"
```

Comment:
Why is this improper? Gelato is a name, right? It should be "Gelato's face" rather than "the Gelato's face."
If you were going to name it "Face of Gelato" it would make sense, but also it shouldn't be capitalized either.